src/main/java/com/powsybl/network/conversion/server/dto/EquipmentInfos.java

```diff
@@ -110,6 +110,14 @@ public static Set<VoltageLevel> getVoltageLevels(@NonNull Identifiable<?> identi
         throw NetworkConversionException.createEquipmentTypeUnknown(identifiable.getClass().getSimpleName());
     }
 
+    public static String getEquipmentTypeName(@NonNull Identifiable<?> identifiable) {
```

Comment:
Knowing we don't manage well hybrid hvdc in gridsuite, I think we should raise an exception if the converter station types are different